docs/queries.rst

```diff
@@ -130,4 +134,4 @@ This will raise 404 errors instead of returning `None`::
 
 Also, if you want to add a description with abort(), you can use it as argument as well.
 
->>> User.query.filter_by(username=username).first_or_404(description='There is no data with {}'.format(username))
\ No newline at end of file
+>>> User.query.filter_by(username=username).first_or_404(description='There is no data with {}'.format(username))
```

Comment:
I'm not entirely sure what changed here...

Reply:
thanks, nothing changed, I'm not sure why it marked a change.